src/components/Admin/ShowcaseCard.vue

```diff
@@ -0,0 +1,92 @@
+<template>
+  <section class="text-center">
+    <q-file class="hidden" multiple ref="artsFileRef" v-model="modelArts" @update:model-value="addArts" />
+    <q-btn flat icon="add_circle_outline" label="Upload Art" rounded @click="onUploadArts" />
+    <div class="items-center q-my-md q-pa-md rounded-borders row shadow-1">
+      <q-spinner v-if="storageStore.isLoading && !modelArts.length" class="q-mx-auto" color="primary" size="3em" />
+      <div v-for="(art, index) in modelArts" class="artImg col-grow q-mx-xs relative-position" :key="index">
+        <q-img class="rounded-borders" fit="contain" :src="art" style="max-height: 10rem" />
+        <q-btn class="trashIcon" color="negative" icon="delete" round size="sm" @click="removeArt(art)" />
+      </div>
+    </div>
+
+    <q-file class="hidden" multiple ref="artistFileRef" v-model="modelArtistPhoto" @update:model-value="addArtistPhoto" />
+    <q-btn flat icon="add_circle_outline" label="Upload Artist Photo" rounded @click="onUploadArtist" />
+    <div class="items-center no-wrap q-my-md q-pa-md rounded-borders row shadow-1">
+      <q-img class="q-mr-md rounded-borders" fit="contain" :src="modelArtistPhoto[0]" style="max-height: 15rem; max-width: 50%" />
+      <q-input autogrow class="col-grow" label="Artist info" v-model="modelArtistInfo" @update:model-value="addArtistInfo" />
+    </div>
+  </section>
+</template>
+
+<script setup>
+import { useErrorStore, useStorageStore } from 'src/stores'
+import { ref } from 'vue'
+
+const props = defineProps(['arts', 'artist', 'date'])
+const emit = defineEmits(['update:arts', 'update:artist'])
+
+const errorStore = useErrorStore()
+const storageStore = useStorageStore()
+
+const artsFileRef = ref(null)
+const artistFileRef = ref(null)
+const modelArts = ref(props.arts)
+const modelArtistInfo = ref(props.artist.info)
+const modelArtistPhoto = ref([props.artist.photo])
+
+function onUploadArts() {
+  artsFileRef.value.pickFiles()
+}
+
+function onUploadArtist() {
+  artistFileRef.value.pickFiles()
+}
+
+async function addArts(files) {
```

Comment:
This function (and onUploadArts) I think need to change. For me, the intuitive process was to pick 1 picture at a time and them keep clicking "+ UPLOAD ART" for every addition I wanted to add. I don't think many users will intuitively know that they need to multi select. This also puts the burden on the user to have all their photos in a single folder on their computer which often isn't the case.

Reply:
Alright, I am gonna fix this in next PR.

This one will be merge to not block the Release 1.3.